### PR TITLE
pythonPackages.MechanicalSoup: 0.4.0 -> 0.7.0

### PR DESCRIPTION
**Pull request by Moredread**
###### Motivation for this change

pythonPackages.MechanicalSoup: 0.4.0 -> 0.7.0

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Moredread** [comment]
@jgillich +

**Comment by Moredread** [comment]
Travis fail seems to be a libnfs issue?

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -11668,11 +11668,11 @@ in {
 
   MechanicalSoup = buildPythonPackage rec {
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: can you move this expression to `python-modules/MechanicalSoup/default.nix`

Comment (Moredread) [reply pkgs/top-level/python-packages.nix]: Done

**Comment by Mic92** [comment]
Thanks!

**Pull request merged**